### ISSUE TITLE
BOSH meltdown doesn't mention deployment

### ISSUE THREAD

**Issue by bkrannich** [cloudfoundry/bosh#1672]
cc: @voelzmo

Currently, BOSH reports a meltdown as:
```
{"id":<redacted>,"parent_id":null,"user":"hm","timestamp":"<redacted>","action":"create",
"object_type":"alert","object_name":"<redacted>","error":null,"task":null,
"deployment":null,"instance":null,
"context_json":"{\"message\":\"We are in meltdown.. Alert @ <redacted>,
severity 1: We are in meltdown.\"}"}
```

Ideally, we'd like to know for which deployment the meltdown happened. As the thresholds set can be percentages, we might run in a situation that one deployment is in meltdown whereas others are not.

**Comment by cppforlife** [comment]
yup, in fact i was just changing prod director to show that as well as a test.

**Comment by cppforlife** [comment]
Story: https://www.pivotaltracker.com/story/show/144906309

**Comment by tylerschultz** [comment]
Addressed by https://github.com/cloudfoundry/bosh/pull/1677

**Comment by tylerschultz** [comment]
This was addressed in PR [#1680](https://github.com/cloudfoundry/bosh/pull/1680/), which superseded #1677. This was released in release version 262.
